lib/mobius/core/command/argument_parser/integer.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Mobius.Command.ArgumentParser.Integer do
+  @moduledoc false
+
+  @behaviour Mobius.Command.ArgumentParser
+
+  @impl true
+  def parse(value) do
+    case Integer.parse(value) do
+      :error -> :error
+      {int, _} -> int
```

Comment:
This will return `5` for the value `"5.0"` which is not exactly what we want. It can be fixed with:

```suggestion
      {int, ""} -> int
      _ -> :error
```

Reply:
Good catch!